core/templates/pages/blog-dashboard-page/blog-dashboard-page.mainpage.html

```diff
@@ -13,20 +13,21 @@
   </head>
   <body>
     <oppia-root>
-      <div ng-controller="Base">
+      <div ng-controller="Base" dir="<[direction]>">
         <oppia-base-content>
           <navbar-breadcrumb>
-            <oppia-blog-dashboard-navbar-breadcrumb>
-            </oppia-blog-dashboard-navbar-breadcrumb>
+            <oppia-blog-dashboard-navbar>
+            </oppia-blog-dashboard-navbar>
           </navbar-breadcrumb>
           <content>
+            <oppia-blog-dashboard-page>
+            </oppia-blog-dashboard-page>
```

Comment:
Only these two components are needed to be downgraded as they are used in an angularjs template. Others are not required to be downgraded.

Reply:
@ashutoshc8101  I just want to confirm that if the blog card component is being used inside blog dashboard component will it have to be downgraded?

Reply:
No

Reply:
Done. Thanks.